### DISCUSSION TITLE
Question: Firebase Credentials Exposed in HTML

### DISCUSSION THREAD

**Discussion by tcmhoang** [vuejs/vuefire#1536] [Questions and Help]
Hi there👋, I noticed the Firebase credentials visible in the rendered HTML in the demo project located in https://nuxt-vuefire-example-blaze.web.app/

![image](https://github.com/vuejs/vuefire/assets/51996720/d4ed8d36-fa66-40c5-b140-f0b1e1fb95f9)
While I couldn't find anything specific in the docs, there should be a way to hide this kind of info from bot scraping.

Any ideas  for hiding these credentials on the client-side? TIA!

**Comment by sugoidesune** [comment]
These credentials are always meant to be public. Security comes from having Rules for DB and storage access.
https://firebase.google.com/docs/rules
